1.0/gradle/vividus-test.gradle

```diff
@@ -132,14 +132,36 @@ task runStories (dependsOn: build, type: RunStoriesTask, group: 'Vividus') {
     }
     ignoreExitValue true
     doLast {
-        if (treatKnownIssuesOnlyAsPassed && executionResult.get().getExitValue() == 1) {
+        if (gradle.taskGraph.hasTask(":${project.name}:verifyRunStatistics"))
+        {
+            logger.warn('\nExit value {} ignored, run statistics will be validated', executionResult.get().getExitValue())
+        }
+        else if (treatKnownIssuesOnlyAsPassed && executionResult.get().getExitValue() == 1) {
             logger.warn('\nKNOWN ISSUES ONLY: All test failures are known issues')
         } else {
             executionResult.get().assertNormalExitValue()
         }
     }
 }
 
+tasks.register('verifyRunStatistics') {
```

Comment:
why not to register it in a regular way?

Reply:
to avoid parameter validation during the configuration phase when task is not used

Reply:
https://docs.gradle.org/current/userguide/task_configuration_avoidance.html#sec:how_do_i_defer_creation